harvester_e2e_tests/scenarios/test_vm_negatives.py

```diff
@@ -82,6 +101,24 @@ def test_delete_vm_while_host_shutdown(self, request, admin_session,
     @pytest.mark.host_management
     def test_vm_after_host_reboot(self, request, admin_session,
                                   harvester_api_endpoints, basic_vm):
+        def _check_vm_instance_nodename():
```

Comment:
maybe we can move this code to utils.py so they can be shared. Perhaps putting it into `lookup_vm_instance()` and add an `ensure_running` parameter?